front/public/locales/en/errors.json

```
@@ -1,11 +1,175 @@
+            },
+            "railjson": {
+                "WrongRailjsonVersionProvided":"Wrong Railjson version provided"
+            }
+        },
+        "layers": {
+            "LayerNotFound":"Layer {{layer_name}} not found. Expected one of {{expected_name",
+            "ViewNotFound":"View {{view_name}} not found. Expected one of {{expected_names}}"
+        },
+        "operation": {
+            "EmptyId":"Empty string id is forbidden",
+            "InvalidPatch":"A Json Patch error occurred",
+            "ModifyId":"Update operation try to modify object id, which is forbidden",
+            "ObjectNotFound":"Object '{{obj_id}}', could not be found in the infrastructure '{{infra_id}}'"
+        },
+        "pagination": {
+            "InvalidPage": "Invalid page number ({{page}})",
+            "InvalidPageSize":"Invalid page size ({{provided_page_size}}), expected an integer 0 < page_size <= {{max_page_size}}"
+        },
+        "pathfinding": {
+            "ElectricalProfilesOverlap": "Electrical Profile overlaps with others",
+            "ElectrificationOverlap":"Electrification {{electrification_id}} overlaps with other electrifications",
+            "InfraNotFound":"Infra {{infra_id}} does not exist",
+            "NotFound":"Pathfinding {{pathfinding_id}} does not exist",
+            "OperationalPointsNotFound":"Operational points do not exist: {{operational_points}}",
+            "RollingStockNotFound":"Rolling stock with id {{rolling_stock_id}} doesn't exist",
+            "TrackSectionsNotFound":"Track sections do not exist: {{track_sections}}"
+        },
+        "postgres": {
+            "Host": "Invalid host '{{hostname}}'",
+            "Password":"Invalid password",
+            "Port":"Invalid port '{{port}}'",
+            "Username":"Invalid username"
+        },
+        "project": {
+            "ImageError":"The provided image is not valid",
+            "ImageNotFound":"Image document '{{document_key}}' not found",
+            "NotFound":"Project '{{project_id}}', could not be found"
+        },
+        "railjson":{
+            "UnsupportedVersion": "Unsupported railjson version"
+        },
+        "redis": {
+            "Url": "Invalid url '{{url}}'"
+        },
+        "rollingstocks": {
+            "BasePowerClassEmpty": "Base power class is an empty string",
+            "CannotCreateCompoundImage":"Impossible to copy the separated image on the compound image",
+            "CannotReadImage": "Impossible to read the separated image",
+            "NameAlreadyUsed":"Name '{{name}}' already used",
+            "NotFound": "Rolling stock '{{rolling_stock_id}}' could not be foun",
+            "RollingStockIsLocked": "RollingStock '{{rolling_stock_id}}' is locked",
+            "RollingStockIsUsed":"RollingStock '{{rolling_stock_id}}' is used"
+        }, 
+        "scenario": {
+            "NotFound": "Scenario not found"
+        },
+        "search" : {
```

Not sure if commons from the linter or my vscode config but when saving the file, it reformats the whole file with less tabulation and it adjusts spaces between `:` and `{`
```suggestion
        "search": {
```

---

I fixed it on vscode, I added a formatter for json files